tests/unit/evaluation/test_base_evaluation.py

```diff
@@ -15,22 +20,79 @@ def test_evaluation_has_accessor_to_fitness_function_eval_count(mocker):
     assert mocked_fit_function.eval_count == 100
 
 
-def test_evaluation_finds_fitness_for_individuals_that_need_it(mocker):
-    population = [mocker.Mock() for _ in range(10)]
+# NOTE (David Randall): multiprocess uses pickle which hates Mock objects,
+# using other objects in place of mocked objects where needed
```

Comment:
pickle and by consequence multiprocessing hate mocked objects. We could either try to get around this using `dill` or an alternative multiprocessing like pathos.multiprocessing or not use mocking in the tests (like I've done here).

Reply:
I'm ok with using dummy objects instead of mocks

Reply:
we might want to write a note in the docs about the requirement for fitness_functions and individuals to be pickle-able in order to use the multiprocessing implementation

Reply:
yeah that sounds good to me, I added a note in the docstring. I'll probably have to make a HPC tutorial in the future so I can include it there too.